### ISSUE TITLE
Option to run just one "should" or one "behavior"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [seratch/scalatest#9]
```
It would be nice to provide an option to run a single test from inside a Spec. 
For instance, if I have this spec:

class ExampleSpec extends Spec {

  describe("A Stack") {

    it("should pop values in last-in-first-out order") (pending)

    it("should throw NoSuchElementException if an empty stack is popped") (pending)
  }
}

and I want to run just the first test, then I should be able to run it from SBT 
console like this:

> test-only ExampleSpec -- -oT="pop values in last-in-first-out order"

This could be based on exact or partial match.
```

Original issue reported on code.google.com by `sr.sa...@gmail.com` on 28 Mar 2012 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
Hi. Good suggestion, and you're not the first to request it. Moreover, a 
solution is pretty much already implemented in trunk. This was done as part of 
the ambitious 2.0 effort, which is focused tools integration. What you can do 
from sbt to IDEs is run just one test or one group of tests within a suite (or 
spec). From sbt or the command line, you can also specify a string that will 
cause a test to run if that string is included in the test name. So you could 
specify "pop" and it would run "should pop values in last-in-first-out order" 
in your example. Oh, it would run the other one too, because that has "popped" 
at the end. So to just run the first one, you might say "pop values".

Basically everything for this is already implemented, except we haven't 
completed the sbt DSL yet.
```

Original comment by `b...@artima.com` on 28 Mar 2012 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
Cool! Thanks for the feedback! Let me know if I can help in anything.
```

Original comment by `sr.sa...@gmail.com` on 28 Mar 2012 at 10:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `billvenn...@gmail.com` on 21 May 2012 at 6:53
- Changed state: **Done**
